res/yaml/locale_EN.yml

```diff
@@ -23,36 +23,36 @@ Headers:
   deadHorseListOtherHeader: "<gold><player>'s dead horses <value> <value>"
   deadHorseListFormat: "<yellow>X<gold> - <yellow><horse><green> <gold>(<yellow><value><gold>)"
   remainingClaimsFormat: "<yellow>(<gold><amount><yellow>/<gold><max><yellow>)"
-  commandWikiHeader: "<green>Wiki : <yellow>https://github.com/Xibalba/ZHorse/wiki/Commands"
-  commandUsageHeader: "<green>Legend : <gold>/command <yellow>mandatoryParams (optionalParams)"
-  commandUsageFormat: "<green>Usage : <value>"
+  commandWikiHeader: "<green>Wiki: <yellow>https://github.com/Xibalba/ZHorse/wiki/Commands"
+  commandUsageHeader: "<green>Legend: <gold>/command <yellow>mandatoryParams (optionalParams)"
+  commandUsageFormat: "<green>Usage: <value>"
   pageNumberFormat: "<yellow>(p. <gold><amount><yellow>/<gold><max><yellow>)"
   horseOptionFormat: "<gold><value><yellow>"
   availableOptionFormat: "<gold><value><yellow>"
 
 Messages:
-  claimsLimitReached: "<yellow>Your claims limit has been reached."
-  claimsLimitReachedOther: "<gold><player><yellow>'s claims limit has been reached."
-  databaseImportFailure: "<yellow>The <gold><value><yellow> data could not be imported into your current database. See logs for details."
-  databaseImportStarted: "<yellow>The <gold><value><yellow> data import has started..."
-  databaseImportSuccess: "<yellow>The <gold><value><yellow> data have been successfully imported into your current database."
-  deadHorsesCleared: "<yellow>All your dead horses have been cleared from the database."
-  deadHorsesClearedOther: "<yellow>All <gold><player><yellow>'s dead horses have been cleared from the database."
+  claimsLimitReached: "<yellow>Your claims limit has been reached."
+  claimsLimitReachedOther: "<gold><player><yellow>'s claims limit has been reached."
+  databaseImportFailure: "<yellow>The <gold><value><yellow> data could not be imported into your current database. See logs for details."
+  databaseImportStarted: "<yellow>The <gold><value><yellow> data import has started..."
+  databaseImportSuccess: "<yellow>The <gold><value><yellow> data have been successfully imported into your current database."
+  deadHorsesCleared: "<yellow>All your dead horses have been cleared from the database."
+  deadHorsesClearedOther: "<yellow>All <gold><player><yellow>'s dead horses have been cleared from the database."
   favoriteAlreadySet: "<yellow>You have already defined <gold><horse><yellow> as favorite horse."
   favoriteAlreadySetOther: "<gold><player><yellow> has already defined <gold><horse><yellow> as favorite horse."
   favoriteEdited: "<gold><horse><yellow> is now your favorite horse."
   favoriteEditedOther: "<gold><horse><yellow> is now the favorite horse of <gold><player><yellow>."
   friendAdded: "<gold><player><yellow> has been added to your friends list."
-  friendAddedReverse: "<gold><player><yellow> has added you in his friends list."
+  friendAddedReverse: "<gold><player><yellow> has added you to their friends list."

```

Comment:
Hey, this looks correct in English. We consider "his" to mean a male. By using "their", this means they could either be of Male or Female gender.

Stating the following, for example: "vueu has added you to their friends list.", looks completely proper!